pkg/netconf/routemap_test.go

```diff
@@ -128,20 +134,20 @@ func Test_importRulesForNetwork(t *testing.T) {
 			input: "testdata/firewall_dmz.yaml",
 			want: []*importRule{
 				{
-					targetVRF:      private.vrf,
-					importVRFs:     []string{inet.vrf, dmz.vrf},
-					importPrefixes: concatPfxSlices(inet.destinations, []importPrefix{publicDefaultNet}, inet.prefixes, dmz.prefixes),
+					TargetVRF:      private.vrf,
+					ImportVRFs:     []string{inet.vrf, dmz.vrf},
+					ImportPrefixes: concatPfxSlices(inet.destinations, []importPrefix{publicDefaultNet}, inet.prefixes, dmz.prefixes),
 				},
 				{
-					targetVRF:      dmz.vrf,
-					importVRFs:     []string{private.vrf, inet.vrf},
-					importPrefixes: concatPfxSlices(private.prefixes, dmz.prefixes, dmz.destinations, inet.prefixes),
+					TargetVRF:      dmz.vrf,
+					ImportVRFs:     []string{private.vrf, inet.vrf},
+					ImportPrefixes: concatPfxSlices(private.prefixes, dmz.prefixes, inet.destinations, inet.prefixes),
```

Comment:
is this a mistake ?

Reply:
This is okay: because we want the default route of the inet vrf to be imported to the DMZ vrf